### ISSUE TITLE
[SPEC] Relayer Open-Market for unlisted Assets

### ISSUE THREAD

**Issue by shekohex** [webb-tools/relayer#406]
### Overview

In the world of cryptocurrency, there are many assets available for trading. However, not all of them are listed on popular exchanges, which makes it difficult for us to fetch the prices of that asset. An open-market for unlisted crypto assets can be very useful in this case.
An open-market for unlisted assets is a feature where the Relayer operator can decide the price of an asset using it's symbol (`tTNT`, `WEBB`, ..etc). The price is determined based on the demand and supply of the asset in question, without the need for bidding or an open-close bookkeeping system, It is really up to the Relayer operator to set the price for any asset, and the user is free to choose which relayer to use and see it fits its needs. A good and honest UI would *optionally* query all the available relayers to get a good quote of the transaction, and sort them by the best price & fees.

### Proposed Config Changes

in the config files we can add a new config option under the global section for asset prices.

For example,

```toml
# Assets in assets.toml
[assets]
tTNT.price = 0.10 # 10 cents
TNT.price = 0.1
# ..etc
```

Also, we should make sure that the asset prices appears when we query the `api/v1/info` endpoint, like the following:

```json
{
  "evm": {},
  "substrate": {},
  "experimental": {},
  "features": {},
  "assets": {
    "tTNT": {
      "price": 0.1
    },
    "TNT": {
      "price": 0.1
    }
  }
}
```

### Using the Configured Asset prices

While querying the Price Oracle API for a price of a cretin token, and we can't find this said token symbol listed on that Oracle, we fallback to read it from our configured assets, However, if we can't find it there either, we should return an error to the user saying `Unsupported  Asset` for example.

### SPEC Checklist

- [ ] Add Support for the `assets` section in the `WebbRelayerConfig`
- [ ] Make sure it is an optional field, defaults to an our pre-configured prices (see below).
- [ ] Change the logic for query the token prices, according to the above instructions.
- [ ] Update the `config/README.md` to add that section to the docs.
- [ ] Update tests if needed, like the typescript configuration. 

### Default Assets

```rust
fn default_assets() -> HashMap<String, UnlistedAsset> {
    HashMap::from_iter([
       (String::from("tTNT"), UnlistedAsset { price: 0.1 }), 
       (String::from("TNT"), UnlistedAsset { price: 0.1 }), 
   ])
}
```

### Related Issues

#404

**Comment by Nutomic** [comment]
By "open-market for unlisted assets" do you mean using something like uniswap for prices like described in #403?

**Comment by shekohex** [comment]
Closed #407
